src/tests/unit/helpers/useFormatMovieLength.spec.js

```diff
@@ -0,0 +1,8 @@
+import { formatMovieLength } from "@/helpers/useFormatMovieLength";
+import { describe, it, expect } from "vitest";
+
+describe("formatMovieLength helper", () => {
+  it("Formats movie length in number of minutes into a string with hours and minutes", () => {
+    expect(formatMovieLength(100)).toBe("1h 40min");
+  });
```

Comment:
I'd only add a second case here where the number of minutes would be 0 (so a multiple of `60`) - because that case it is not known from this spec if it would be something like `1h` or `2h` or `1h 0min` 😄

Reply:
Yeah, you got me, that's actually an edge case I haven't considered while writing the function 😅
I'm on it!